keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -50,9 +50,11 @@ private fun ClusterSpec.resolveLaunchConfiguration(region: SubnetAwareRegionSpec
     ebsOptimized = checkNotNull(overrides[region.name]?.launchConfiguration?.ebsOptimized
       ?: defaults.launchConfiguration?.ebsOptimized),
     iamRole = checkNotNull(overrides[region.name]?.launchConfiguration?.iamRole
-      ?: defaults.launchConfiguration?.iamRole),
+      ?: defaults.launchConfiguration?.iamRole
+      ?: LaunchConfiguration.defaultIamRoleFor(moniker.app)),
     keyPair = checkNotNull(overrides[region.name]?.launchConfiguration?.keyPair
-      ?: defaults.launchConfiguration?.keyPair),
+      ?: defaults.launchConfiguration?.keyPair
+      ?: LaunchConfiguration.defaultKeyPairFor(locations.account, region.name)),
```

Comment:
I'd still like to see the netflixism that is `defaultKeyPairFor` removed from keel.

Reply:
(Can happen in a follow-up PR though)

Reply:
I added https://github.com/spinnaker/keel/issues/604 to follow-up.